timemachine/fe/single_topology_v3.py

```diff
@@ -269,16 +270,32 @@ def setup_end_state(ff, mol_a, mol_b, core, a_to_c, b_to_c):
     mol_c_torsion_idxs = mol_c_proper_idxs + mol_c_improper_idxs
     mol_c_torsion_params = mol_c_proper_params + mol_c_improper_params
 
-    bond_potential = potentials.HarmonicBond(mol_c_bond_idxs).bind(mol_c_bond_params)
-    angle_potential = potentials.HarmonicAngle(mol_c_angle_idxs).bind(mol_c_angle_params)
-    torsion_potential = potentials.PeriodicTorsion(mol_c_torsion_idxs).bind(mol_c_torsion_params)
+    # canonicalize bonds
```

Comment:
```suggestion
```

(or add `# canonicalize angles`, `# canonicalize torsions` etc. comments to subsequent blocks)

Reply:
willfix